[2022-02-03 20:54:23,397] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-03 20:54:23,461] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-03 20:54:23,463] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:54:23,465] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 20:54:23,467] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:54:23,591] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-03-01 00:00:00+00:00
[2022-02-03 20:54:23,620] {standard_task_runner.py:52} INFO - Started process 7085 to run task
[2022-02-03 20:54:23,645] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_5', 'download_dataset_task', 'scheduled__2020-03-01T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmp_li3hy32', '--error-file', '/tmp/tmp6g5n1al4']
[2022-02-03 20:54:23,657] {standard_task_runner.py:77} INFO - Job 62: Subtask download_dataset_task
[2022-02-03 20:54:23,903] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:54:24,158] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_5
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-01T00:00:00+00:00
[2022-02-03 20:54:24,165] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:54:24,168] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:54:24,191] {subprocess.py:85} INFO - Output:
[2022-02-03 20:54:24,240] {subprocess.py:89} INFO - /tmp/***tmpp487wton
[2022-02-03 20:54:24,245] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:54:24,326] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_5, task_id=download_dataset_task, execution_date=20200301T000000, start_date=20220203T205423, end_date=20220203T205424
[2022-02-03 20:54:24,413] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:54:24,668] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
